examples/src/main/scala/examples/hybrid/history/HybridHistory.scala

```diff
@@ -407,14 +424,26 @@ class HybridHistory(val storage: HistoryStorage,
     def in(m: HybridBlock): Boolean = loserChain.exists(s => s sameElements m.id)
 
     val winnerChain = chainBack(forkBlock, in, limit).get.map(_._2)
-    val i = loserChain.indexWhere(id => id sameElements winnerChain.head)
+    val i = loserChain.indexWhere { id =>
+      winnerChain.headOption match {
+        case None                  => false
+        case Some(winnerChainHead) => id sameElements winnerChainHead
+      }
+    }
```

Comment:
I made this change here as I think it is the intended semantics. Should I take it back?

Reply:
I also think this is the intended semantics.